python/pyspark/ml/image.py

```diff
@@ -201,8 +243,9 @@ def readImages(self, path, recursive=False, numPartitions=-1,
         .. versionadded:: 2.3.0
         """
 
-        spark = SparkSession.builder.getOrCreate()
-        image_schema = spark._jvm.org.apache.spark.ml.image.ImageSchema
+        ctx = SparkContext.getOrCreate()
```

Comment:
minor comment: the change before was only two lines whereas this is 3 lines - I think the previous change was better, what is the advantage of the new code?

Reply:
Good catch. Looks like this was caused by auto-rebasing onto the latest changes. My original change was only to replace _active_context with getOrCreate() but it was clearly made obsolete in the meantime. I'll just remove it.